warehouse/templates/manage/token.html

```diff
@@ -90,7 +90,8 @@ <h2>Add another token</h2>
           <option disabled selected value="scope:unspecified">Select scope...</option>
```

Comment:
We should probably make the `selected` attribute here conditional based on what `create_macaroon_form.token_scope.data` is.